happi/loader.py

```diff
@@ -201,6 +205,12 @@ def create_arg(arg):
         except Exception:
             logger.warning("Unable to attach metadata dictionary to device")
 
+        if run_post_attach_hook and hasattr(obj, 'post_happi_md'):
```

Comment:
I like the method name except that it sounds super similar to the happi item's `post` method. Probably not an issue though. https://github.com/pcdshub/happi/blob/c46a5e9e81741b01e5f98629cae460662be33d1c/happi/item.py#L367

Reply:
`after_happi_md` doesn't have the same ring to it..
`happi_md_hook`?  eh

Reply:
Yeah idk it shouldn't be confusing at all to people who haven't poked around with happi internals
I blame English for ascribing too many meanings to "post"